frontend/src/metabase-lib/lib/Question.ts

```diff
@@ -538,16 +538,20 @@ class QuestionInner {
     return pivot(this, breakouts, dimensions) || this;
   }
 
-  drillUnderlyingRecords(values: DimensionValue[], column?: Column): Question {
+  drillUnderlyingRecords(
+    dimensions: DimensionValue[],
+    column?: Column,
+  ): Question {
     let query = this.query();
     if (!(query instanceof StructuredQuery)) {
       return this;
     }
 
-    query = values.reduce(
-      (query, { value, column }) => drillFilter(query, value, column),
-      query,
-    );
+    dimensions.forEach(({ value, column }) => {
+      if (column.source !== "aggregation") {
```

Comment:
This is from https://github.com/metabase/metabase/blob/e8526c757ee9cb1cd883543ebd8170e531a63634/frontend/src/metabase/visualizations/lib/apply_tooltips.js#L211 AFAIK we should never get `query-transform` columns here, so I only checked for aggregation columns. 

The thing is that `apply_tooltips` removes aggregation dimensions from one-stages queries and it doesn't do for other cases. It seems too risky to modify `apply_tooltips`, that's why I only changed it here.